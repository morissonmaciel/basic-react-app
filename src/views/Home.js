import React from "react";
import ReactLogo from "../assets/images/react-logo.png";
import {PopOver} from "../components/PopOver";
import {ContentView} from "./Home.styled";
import {ReactComponent as LampSvg} from "../assets/images/lamp.svg";

export const Home = () => {
  return (
    <ContentView>
      <PopOver>
        <img src={ReactLogo} width={128}/>
      </PopOver>
      <p className="centered">
        This is a lightweight React app based on a simple and faster structure.</p>
      <br/>
      <LampSvg width={24} height={24}/>
      <p className="centered">
        Start a new project coping contents from repository with command:</p>
      <pre className="centered">
                $ git clone https://github.com/morissonmaciel/basic-react-app.git [project_name]</pre>
    </ContentView>
  );
};