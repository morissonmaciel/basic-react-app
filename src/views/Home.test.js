import React from "react";
import { shallow } from "enzyme";
import { Home } from "./Home";

it("Should render Home correctly", () => {
    const wrapper = shallow(<Home/>);
    expect(wrapper).toMatchSnapshot();
});
