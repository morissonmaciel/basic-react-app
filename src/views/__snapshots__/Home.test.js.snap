// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Home correctly 1`] = `ShallowWrapper {}`;
